projects/gdal/build.sh

```diff
@@ -15,10 +15,28 @@
 #
 ################################################################################
 
+I386_PACKAGES="zlib1g-dev:i386 libexpat-dev:i386 liblzma-dev:i386 \
+              libxerces-c-dev:i386 libpng12-dev:i386 libgif-dev:i386 \
+              libwebp-dev:i386 libicu-dev:i386 libnetcdf-dev:i386 \
+              libssl-dev:i386 libsqlite3-dev:i386"
+X64_PACKAGES="zlib1g-dev libexpat-dev liblzma-dev \
+              libxerces-c-dev libpng12-dev libgif-dev \
+              libwebp-dev libicu-dev libnetcdf-dev \
+              libssl-dev libsqlite3-dev"
+
+if [ "$ARCHITECTURE" = "i386" ]; then
+    apt-get remove -y *-dev:amd64 || echo "go on"
```

Comment:
What is this for ? Would prefer these two apt-get remove calls removed. Unsure what sort of instability it can create.

Reply:
I did that because I wasn't sure if ./build.sh could not be called several times with different arguments on a previous image, and if so if there are -amd64 packages already installed when buildling for i384, uninstalling them seemed safer. If you confirm that buil.sh is always called on a fresh Dockerfile image, then yes i'll remove then

Reply:
@oliverchang can confirm (i think answer is yes). if yes, please remove these 2 lines and commit.

Reply:
Yes, build.sh should not be called more than once on the same container instance.

Reply:
Thanks @oliverchang , removed those and committed.